.github/workflows/ci-continuous-integration.yml

```diff
@@ -46,6 +46,40 @@ jobs:
       - name: Run unit tests
         run: |
           ci-scripts/unit-tests.sh          
+  ssr_tests:
+    name: CI - SSR tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Cache node_modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          path: | 
+            node_modules
+            projects/storefrontapp-e2e-cypress/node_modules
+            ~/.cache/Cypress
+          key: nodemodules-${{ github.event.pull_request.base.sha }}
+          restore-keys: nodemodules-${{ github.event.pull_request.base.sha }}
+      - name: Package installation
+        run: npm ci
+      - name: Build SSR Server
+        run: npm run build:libs && npm run build && npm run build:ssr
+      - name: Start SSR Server
+        run: npm run serve:ssr > ssr-e2e/ssr.log &
+      - name: Expose dirs    
+        run: |
+          ls ./ &&
+          ls ../ &&
+          ls ./ssr-e2e
+      - name: Run SSR tests
+        run: npm run test:ssr
```

Comment:
Now the long-running SSR server instance is shared among many test specs.
This is prone to specs interfering with each other via the shared SSR server instance, which may lead to unexpected false-positives and false-negatives, depending on the behavior of SSR server (which is a subject of implementation, not tests).

If possible, it would be good to isolate specs, by starting and killing SSR server for each spec separately - so no server instance is shared among 2 specs. I'd be happy to learn if we can do it without increasing significantly the total time of running all the specs.

If the total execution time of e2e is big, we might consider to NOT run them on every PR, but only on `develop` branch (like other test suites which are not crucial or time-costly).